docker/Dockerfile

```diff
@@ -156,15 +150,16 @@ COPY --chown=$user:$user .pre-commit-config.yaml $HOME/code/garage
 RUN git init && \
   pip install pre-commit && \
   pre-commit install && \
-  pre-commit install-hooks
+  pre-commit install-hooks && \
+  rm -r $HOME/.cache/pip
 
 # Install deps (but not the code)
-# Note that we don't delete the pip cache for dev builds
 RUN rm -rf $VIRTUAL_ENV && \
   python -m virtualenv $VIRTUAL_ENV && \
   pip install --upgrade pip setuptools wheel && \
   pip install .[all,dev] && \
-  rm $HOME/.mujoco/mjkey.txt
+  rm $HOME/.mujoco/mjkey.txt && \
+  rm -r $HOME/.cache/pip
```

Comment:
does this make https://github.com/rlworkgroup/garage/pull/1576/files#diff-ebacf6f6ae4ee68078bb16454b23247dR170 run slower?

Reply:
Line 170 should find everything in venv and wouldn't have to go to pip cache. It wasn't affected in my local test